lib/ares.rb

```diff
@@ -18,6 +20,12 @@ def client
     # @see Client::Standard#call
     # @return [Responses::StandardResponse::Record]
     def standard(options)
+      if options[:ico]
```

Comment:
I think it would be better to move this logic into separate method like:

```ruby
def standard(options)
  validate_ico_format
  response = client.call(options)
  ...
end
```